src/docs/product/projects/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Projects
+sidebar_order: 1
+description: "View all the projects you're a member of, then dive into their details quickly."
+---
+A _Project_ connects the error and transaction events in your application to the team members assigned to that project.
+
+You can create a project for a particular language or framework, and typically do so when you first start using Sentry, though you can add projects as you need. For example, you might have separate projects for your API server and frontend client. Setting up multiple projects to reflect your application landscape with finer granularity will go a long way in helping with visibility and triage.
+
+Also, when displaying the **Issue Stream** and **Discover** views, the top-level filter bar considers the projects you are a member of by default. This way, you are looking at information that is immediately relevant to your work.
+
+## Project Homepage
+
+The project homepage provides a high-level overview of all your projects, including:
+
+- All projects of which you're a member
+- A snapshot of both errors and transactions
+- Team members for each project
+- Deploys, if you have configured your SDK to [provide a release identifier](/platform-redirect/?next=/configuration/releases/)
```

Comment:
Providing releases is only half of it. Additionally, you have to tell Sentry about the deploys. Unfortunately, we do not have extensive documentation on deploys yet. The best I found was https://docs.sentry.io/product/cli/releases/#creating-deploys.

Reply:
Ok. I will edit to include deploys as well. 

Should we add a GH issue to track that we need to develop more deploy content?